cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -367,8 +384,10 @@ private void createConfigTable(String configTableName) throws IOException {
   @Override
   public void dropAllInNamespace(String namespaceId) throws Exception {
```

Comment:
this change is a little larger, so will do as separate PR: https://issues.cask.co/browse/CDAP-1483